t/35-script_clone_job_suse.t

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+use Test::Most;
+use Test::Warnings ':report_warnings';
+
+use FindBin;
+use lib "$FindBin::Bin/lib", "$FindBin::Bin/../external/os-autoinst-common/lib";
+use OpenQA::Test::TimeLimit '6';
+use Mojo::Base -signatures;
+use Test::MockModule;
+use OpenQA::Script::CloneJobSUSE;
+use Mojo::URL;
+
+# define fake client
+{
+    package Test::FakeLWPUserAgentMirrorResult;
+    use Mojo::Base -base, -signatures;
+    has is_success => 1;
+}
+
+{
+    package Test::FakeLWPUserAgent;
+    use Mojo::Base -base, -signatures;
+    has is_validrepo => 1;
+    sub get ($self, $url) {
+        my %res
+          = (!$self->is_validrepo)
+          ? (is_success => 0)
+          : ();
+        Test::FakeLWPUserAgentMirrorResult->new(%res);
+    }
```

Comment:
```suggestion
    sub get ($self, $url) { Test::FakeLWPUserAgentMirrorResult->new(is_success => $self->is_validrepo) }
```

Reply:
Modified the code as suggested. Thank you.